### ISSUE TITLE
Multilingual Pages Converter solution does not build right away - has missing references

### ISSUE THREAD

**Issue by heinrich-ulbricht** [pnp/sp-dev-solutions#406]
#### Category
- [ ] Question
- [x] Bug
- [ ] Enhancement

#### Expected or Desired Behavior

I expect that the _MultilingualPagesConverter_ solution can be built right away. Or that the installation guide states which references need to be set up/installed manually (which I'll happily do).

#### Observed Behavior

The solution cannot be built. Those references are missing:

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0246	The type or namespace name 'Newtonsoft' could not be found (are you missing a using directive or an assembly reference?)	MultilingualPagesConverter	C:\00_Repos\sp-dev-solutions\solutions\MultilingualPages\Converter\MultilingualPagesConverter\Program.cs	2	Active
Error	CS0234	The type or namespace name 'SharePoint' does not exist in the namespace 'Microsoft' (are you missing an assembly reference?)	MultilingualPagesConverter	C:\00_Repos\sp-dev-solutions\solutions\MultilingualPages\Converter\MultilingualPagesConverter\Program.cs	1	Active
Error	CS0246	The type or namespace name 'OfficeDevPnP' could not be found (are you missing a using directive or an assembly reference?)	MultilingualPagesConverter	C:\00_Repos\sp-dev-solutions\solutions\MultilingualPages\Converter\MultilingualPagesConverter\Program.cs	3	Active
Error	CS0246	The type or namespace name 'ListItem' could not be found (are you missing a using directive or an assembly reference?)	MultilingualPagesConverter	C:\00_Repos\sp-dev-solutions\solutions\MultilingualPages\Converter\MultilingualPagesConverter\Program.cs	21	Active
Error	CS0246	The type or namespace name 'ListItem' could not be found (are you missing a using directive or an assembly reference?)	MultilingualPagesConverter	C:\00_Repos\sp-dev-solutions\solutions\MultilingualPages\Converter\MultilingualPagesConverter\Program.cs	22	Active
Error	CS0246	The type or namespace name 'ListItem' could not be found (are you missing a using directive or an assembly reference?)	MultilingualPagesConverter	C:\00_Repos\sp-dev-solutions\solutions\MultilingualPages\Converter\MultilingualPagesConverter\Program.cs	23	Active
Error	CS0246	The type or namespace name 'ListCollection' could not be found (are you missing a using directive or an assembly reference?)	MultilingualPagesConverter	C:\00_Repos\sp-dev-solutions\solutions\MultilingualPages\Converter\MultilingualPagesConverter\Program.cs	24	Active
Error	CS0305	Using the generic type 'List<T>' requires 1 type arguments	MultilingualPagesConverter	C:\00_Repos\sp-dev-solutions\solutions\MultilingualPages\Converter\MultilingualPagesConverter\Program.cs	25	Active
Error	CS0246	The type or namespace name 'ClientContext' could not be found (are you missing a using directive or an assembly reference?)	MultilingualPagesConverter	C:\00_Repos\sp-dev-solutions\solutions\MultilingualPages\Converter\MultilingualPagesConverter\Program.cs	331	Active
Error	CS0246	The type or namespace name 'ClientContext' could not be found (are you missing a using directive or an assembly reference?)	MultilingualPagesConverter	C:\00_Repos\sp-dev-solutions\solutions\MultilingualPages\Converter\MultilingualPagesConverter\Program.cs	380	Active
Error	CS0246	The type or namespace name 'ListCollection' could not be found (are you missing a using directive or an assembly reference?)	MultilingualPagesConverter	C:\00_Repos\sp-dev-solutions\solutions\MultilingualPages\Converter\MultilingualPagesConverter\Program.cs	380	Active
Error	CS0246	The type or namespace name 'ClientContext' could not be found (are you missing a using directive or an assembly reference?)	MultilingualPagesConverter	C:\00_Repos\sp-dev-solutions\solutions\MultilingualPages\Converter\MultilingualPagesConverter\Program.cs	461	Active
```

#### Steps to Reproduce

Follow the instructions: clone the repo, navigate to the _Converter_ folder, open solution, build: errors.

Note that I clicked "Restore NuGet Packages" on the solution since I thought this might be the culprit. But apparently they are all ready:
![image](https://user-images.githubusercontent.com/3469970/83157739-3b984f00-a104-11ea-858b-eb46249799e6.png)

Microsoft Visual Studio Professional 2019
Version 16.5.2
Windows 10

@juliemturner (I hope this is the same handle as @jfj1997 which is listed as author...)

**Comment by juliemturner** [comment]
I can't speak to Visual Studio Professional, that IDE is not something I use for SharePoint Framework development and it doesn't use NuGet packages it uses NPM packages.

At this point though, Microsoft has GA's their multilingual pages solution which pretty much deprecates this interim solutions... so might I point you to using the new [OOB feature](https://support.office.com/en-us/article/create-multilingual-communication-sites-pages-and-news-2bb7d610-5453-41c6-a0e8-6f40b3ed750c) instead or was there something specific you were looking for.

**Comment by heinrich-ulbricht** [comment]
I mean this good ol' solution: https://www.github.com/pnp/sp-dev-solutions/tree/master/solutions%2FMultilingualPages%2FConverter%2FMultilingualPagesConverter%2FMultilingualPagesConverter.sln

@juliemturner You are listed as author on the readme page. Or I'm using this new Android GitHub app wrong...

**Comment by juliemturner** [comment]
Oh, sorry I missed the "converter" part... um, yeah I did develop it... hmm... it shouldn't have that issue, I will put it on my backlog to look at it but I can't promise it'll be soon.

**Comment by heinrich-ulbricht** [comment]
I gave it a shot and it seems like Visual Studio got confused by the existing `packages` folder. Removing it before opening the solution solves the problem for me and Visual Studio will be able to restore the NuGet packages properly (which it wasn't before). Building succeeds.

I'll create a pull request that removes the `packages` folder.

(Note: Calling `Update-Package -reinstall` instead did _not_ work for me as this failed midway with some COM error.)

**Comment by juliemturner** [comment]
Fixed by PR #411
